mooncake-common/etcd/etcd_wrapper.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+/*
+#include <stdint.h> 
+*/
+import "C"
+
+import (
+	"context"
+	"time"
+
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"unsafe"
+)
+
+var globalClient *CClient
```

Comment:
```suggestion
var client_id uint64
var globalClient map[uint64]*CClient
```

Maybe use a map could be more robust, and return a client_id in NewEtcdClient.